modules/asset-registry/Cargo.toml

```diff
@@ -17,21 +17,28 @@ primitives = { workspace = true }
 xcm = { workspace = true }
 
 module-support = { workspace = true }
+module-evm = { workspace = true }
+serde_json = { workspace = true }
+hex = { workspace = true }
```

Comment:
The new benchmark requires importing the associated module as a dependency, which I’m not sure is what we want.
https://github.com/AcalaNetwork/Acala/blob/a6bb42cab4312a2d3c935abcc741b2ca6f3dc93c/runtime/mandala/src/benchmarking/dex_oracle.rs#L51-L52
the dex-oracle need to import the `module-currencies`, `module-dex`, `pallet-aura`, `pallet-balances`, `pallet-timestamp`, `sp-consensus-aura`,  it feels too bulky

Reply:
can we mock it?